CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ if(EXIV2_BUILD_EXIV2_COMMAND)
       WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
       COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose lens_tests
     )
+    # Repeat lens test with a DE locale that uses "," as float delimiter
```

Comment:
Is it possible to create one specific test instead of repeating the whole suite?

Reply:
This gives better coverage than a single file would. This adds O(10s) to the test time so I don't think it's a problem.

Reply:
Sure, but why stop there then? ;) Either we have a targeted test for this bug, or we cover really _everything_ in a different locale? We should probably open a TODO issue...

Reply:
As you say, there could more stuff like this not related just to lens detection: https://github.com/Exiv2/exiv2/blob/19887c8572e13b2e9ead9f361d707bd7c9dccf9e/src/convert.cpp#L684

Reply:
> Sure, but why stop there then? 

Because I've got very limited, actually near zero, time to work on exiv2. And this was the most coverage I could get without major work. 

I agree that broader testing should be done. But I won't have the time to tackle that. I agree that there should be an issue to track this.